src/studio/editors/code.c

```diff
@@ -1157,6 +1157,19 @@ static void doTab(Code* code, bool shift, bool crtl)
     else inputSymbolBase(code, '\t');
 }
 
+// Add an "end" keyword, and put the cursor in the line between.
+static void newLineWithEnd(Code* code)
```

Comment:
TIC-80 also supports other scripts langs like JS, Squirrel, Wren, Fennel, and Moonscript. Please add the function "end" depending on lang, you can define it in `tic_script_config` struct (for example `const char* end = tic_core_script_config(tic)->functionEnd;`).
Thank you.